Xamarin.Forms.Platform.iOS/Shapes/RectangleRenderer.cs

```diff
@@ -11,6 +11,8 @@ namespace Xamarin.Forms.Platform.MacOS
 {
     public class RectangleRenderer : ShapeRenderer<Rect, RectView>
     {
+        const double MaximumRadius = 0.5d;
```

Comment:
Why this value?

Reply:
The maximum value for the cornerHeight and cornerWidth using `AddRoundedRect` is the middle of the Height and Width.
https://developer.apple.com/documentation/coregraphics/cgmutablepath/2427144-addroundedrect

Added a comment in the code.

Reply:
Love the comment!